Framework/CurveFitting/src/Algorithms/IqtFit.cpp

```diff
@@ -164,19 +167,27 @@ double IqtFit<QENSFitSimultaneous>::getStartX(std::size_t index) const {
   return QENSFitSimultaneous::getProperty("StartX" + getPropertySuffix(index));
 }
 
-template <typename Base>
-double IqtFit<Base>::getStartX(std::size_t /*unused*/) const {
-  return Base::getProperty("StartX");
+template <> double IqtFit<QENSFitSequential>::getStartX(std::size_t i) const {
+  std::vector<double> startX = QENSFitSequential::getProperty("StartX");
+  if (startX.size() == 1) {
+    return startX[0];
+  } else {
+    return startX[i];
+  }
```

Comment:
Should we check if `i >= startX.size()` ?

Reply:
this part of Iqtfit inherits from QNESSequential and uses it's validation, that means that startX must either contain a single value, or the same number of values as the size of the input workspace. So `i` can not be greater than startX.size()